### ISSUE TITLE
Exception on finding linux distro if /etc/ has folders that end with 'release'

### ISSUE THREAD

**Issue by serzhshakur** [bonigarcia/webdrivermanager#129]
It may happen on Linux that /etc/ has not only files with 'release' postfix, but also folders. For example I have Linux Mint Cinnamon v18.1 and in my case there's folder /etc/upstream-release/. So I receive `java.io.FileNotFoundException: /etc/upstream-release (Is a directory)`. This could be solved by filtering out only files in [BrowserManager.java:444](https://github.com/bonigarcia/webdrivermanager/blob/master/src/main/java/io/github/bonigarcia/wdm/BrowserManager.java#L444) in the following way:

```
File fileList[] = new File[0];
    if (dir.exists()) {
		fileList = dir.listFiles(new FileFilter() {
			public boolean accept(File file) {
				return file.isFile() && file.getName().endsWith("-release");
		}
	});
}
```

**Comment by serzhshakur** [comment]
would fix it myself but have no rights to create branch

**Comment by serzhshakur** [comment]
as it turned out it's already fixed in release 1.6.2
